<ui version="4.0" >
 <class>ImportBinary</class>
 <widget class="QDialog" name="ImportBinary" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Import from binary file</string>
  </property>
  <layout class="QFormLayout" name="formLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>File:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLineEdit" name="File_" />
     </item>
     <item>
      <widget class="QPushButton" name="Browse_" >
       <property name="text" >
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Additional tags:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="AdditionalTags_" >
     <property name="text" >
      <string>imported</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Feeds:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QTreeWidget" name="FeedsToImport_" >
     <column>
      <property name="text" >
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>URL</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Items</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QDialogButtonBox" name="ButtonBox_" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonBox_</sender>
   <signal>accepted()</signal>
   <receiver>ImportBinary</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox_</sender>
   <signal>rejected()</signal>
   <receiver>ImportBinary</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
